ext/date/tests/DateTime_verify.phpt

```diff
@@ -27,7 +27,7 @@ object(ReflectionClass)#%d (1) {
   string(8) "DateTime"
 }
 ..and get names of all its methods
-array(18) {
+array(19) {
```

Comment:
I was asking the same question myself, but just cowardly changed the expectf, because I didn't want to remove tests in the PR.